### ISSUE TITLE
require a character @ to begin valid signature

### ISSUE THREAD

**Issue by axelpale** [axelpale/yamdog#33]
In practice it is necessary to separate doc blocks from other matching comment blocks. The comments often discuss the concepts of the program, thus the text contains signatures by accident. If the signature line is prefixed with @ or other special character then the accident probability drops. The default @ is a respective choice as the jsdoc uses @ for all the keywords.

**Comment by axelpale** [comment]
Either release 1.7 with an option to force the character OR release 2.0 with @ being the default.

**Comment by axelpale** [comment]
Maybe replace earmark meaning and remove earmark from the output. For example the earmark `@`. After that, all comment blocks that begin with `@` are included. But what happens to multiple earmarks/names feature?

**Comment by axelpale** [comment]
Example:
```
const yamdog = require('yamdog')
const path = require('path')
yamdog.generate({
  // Where to start collecting comment blocks
  entry: path.resolve(__dirname, '../'),
  // Where to generate
  output: path.resolve(__dirname, 'API.md'),
  // Earmark; include comment blocks that begin with this string
  earmark: '@',
  // Names; include comment blocks with these names (string after the earmark)
  names: ['yamdog'],
  // Main title of the document
  title: 'Doghouse API Documentation',
  // Introduction; the initial paragraph
  intro: 'Welcome to Doghouse API documentation.',
  // Decorators; a customizable features to pimp yo docs
  decorators: [
    yamdog.decorators.alphabetical(), // render in alphabetical order
    yamdog.decorators.toc() // insert tables of contents
  ]
})
```

**Comment by axelpale** [comment]
Fixed in v2.0.0 as the example above
